src/components/controls/VersionControl/VersionControl.js

```diff
@@ -0,0 +1,32 @@
+// Controlled selector that allows a user to switch between CMIP5 and
+// CMIP6 datasets. 
+
+import React from 'react';
+import Select from 'react-select';
+
+export default function VersionControl( { onSelect, selected } ) {
+	// this is all just hard-coded, it's complocated to
+	// get it from the API.
+	const items = [
+		{"value": "CMIP6", "label": "CMIP6+CWEC2020"},
+		{"value": "CMIP5", "label": "CMIP5+CWEC2016"}
+	];
+	
+	const label = "Future-shifted Dataset";
+	
+	const value = () => {
+		return items.find((o) => o["value"] === selected)
+	};
+	
+	return (
+		<div>
+		{label}
```

Comment:
Is there a reason not to put the label string straight in here rather than using a const? Alternatively to make it a prop with this as default?

Reply:
Also, let's consider making the label bold. That would match with the table headings and call it out more clearly.